### DISCUSSION TITLE
[CN DateTimeV2] Traditional Chinese support?

### DISCUSSION THREAD

**Discussion by Voileexperiments** [microsoft/Recognizers-Text#1844] [General]
The pattern for `CN DateTimeV2` is named `Chinese` but it's actually for simplified Chinese only, so it's misleading. If it's intended then the naming is inconsistent: Language codes like `EN` are the first part (like `en-us` and `en-gb`) while `CN` is the second part (as in `zh-cn` and `zh-tw`). Also, Chinese, if not specified, includes both Simplified Chinese and Traditional Chinese (HK and TW).

Adding support for Traditional Chinese based on the existing Simplified Chinese file doesn't seem too difficult: most things can be converted by a Simplified Chinese -> Traditional Chinese converter, and the rest is adding the Traditional Chinese only phrases.

**Comment by tellarin** [comment]
@Voileexperiments, you are right that the current implementation mostly supports ZH-CN (https://github.com/microsoft/Recognizers-Text#supported-entities-across-cultures).
However the goal is for the same Chinese model to be able to cover both Simplified and Traditional variants. This is already the case for a few cases, but there's much more to do.
Would you be interested in helping improve support for Traditional Chinese?
